### ISSUE TITLE
`PackedBundle` with single element gives assignment overlap

### ISSUE THREAD

**Issue by oletf** [SpinalHDL/SpinalHDL#1615]
Hi,

Just stumbled on this error:
```scala
   case class TestBundleOne() extends PackedBundle {
        val field = SInt(16 bits) default(0x1234)
      }
   case class TestBundleTwo() extends PackedBundle {
        val field1 = SInt(8 bits) default(0x34)
        val field2 = SInt(8 bits) default(0x12)
      }

val test1 = TestBundleOne().packed // ASSIGNMENT OVERLAP
val test2 = TestBundleTwo().packed // works as expected
```

> spinal.lib.PackedBundle.$anonfun$packed$3([PackedBundle.scala:95](https://github.com/SpinalHDL/SpinalHDL/blob/fb2a8df445631b3da65e84f7ee254d0c4b4c4c67/lib/src/main/scala/spinal/lib/PackedBundle.scala#L95))
    spinal.lib.PackedBundle.$anonfun$packed$3$adapted([PackedBundle.scala:81](https://github.com/SpinalHDL/SpinalHDL/blob/fb2a8df445631b3da65e84f7ee254d0c4b4c4c67/lib/src/main/scala/spinal/lib/PackedBundle.scala#L81))
    spinal.lib.PackedBundle.packed(PackedBundle.scala:81)

Seems like packing of a Bundle with a single element breaks the packing...

Did I miss something in the use case ?

Thanks forward!

**Comment by dokleina** [comment]
Hi,

This should have been fixed with https://github.com/SpinalHDL/SpinalHDL/pull/1457. Are you able to test it with the latest from `dev`?

**Comment by oletf** [comment]
Hi!

Confirm it's fixed with [1.11.0](https://github.com/SpinalHDL/SpinalHDL/releases/tag/v1.11.0) which has #1457 already merged in ! 
Thanks!
